test/extended/build_context_dir_test.go

```diff
@@ -0,0 +1,76 @@
+// +build integration,extended,!no-etcd,!no-docker,docker
```

Comment:
should it just be extended?  these tests aren't run when hack/test-integration.sh is run (because they're in a different package), so tagging them as integration is a bit confusing.